cate/webapi/start.py

```diff
@@ -72,12 +73,21 @@
 
 
 # noinspection PyAbstractClass
-class WebAPIVersionHandler(WebAPIRequestHandler):
+class WebAPIInfoHandler(WebAPIRequestHandler):
     def get(self):
+        workspace_manager_mode = "unknown"
+        if self.application.workspace_manager:
+            def camelcase_to_underscore(inp: str):
+                return re.sub(r'(?<=[a-z])[A-Z]|[A-Z](?=[^A-Z])', r'_\g<0>', inp).lower().strip('_')
+
+            name = self.application.workspace_manager.__class__.__name__
+            workspace_manager_mode = camelcase_to_underscore(name.replace("WorkspaceManager", ""))
```

Comment:
```suggestion
            user_root_mode = self.application.root_dir is not None
```

Something like this is needed.